Formula/arangodb.rb

```diff
@@ -16,14 +16,33 @@ class Arangodb < Formula
   depends_on :macos => :yosemite
   depends_on "openssl"
 
+  # see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87665
   fails_with :gcc do
     build 820
     cause "Generates incorrect code"
   end
 
+  # the ArangoStarter is in a separate github repository;
+  # it is used to easily start single server and clusters
+  # with a unified CLI
+  resource "starter" do
+    url "https://github.com/arangodb-helper/arangodb.git",
+      :revision => "ca2ddf942ba63c47bbccdc47fd362377b8c88b19"
+  end
+
   def install
     ENV.cxx11
 
+    resource("starter").stage do
+      ENV.append "GOPATH", Dir.pwd + "/.gobuild"
+      system "make", "deps"
+      # use commit-id as projectBuild
+      system "go", "build", "-ldflags", "-X main.projectVersion=0.14.0 -X main.projectBuild=ca2ddf942ba63c47bbccdc47fd362377b8c88b19",
```

Comment:
@Homebrew/core is there a way to reference the resource commit here? That seems like a clearer solution.

Reply:
Use `git rev-parse` or an actual `git` checkout instead of using a tarball. Agreed that hardcoding this is non-ideal.

Reply:
@SMillerDev @MikeMcQuaid changed to "git rev-parse"